microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -196,6 +197,25 @@ For reference information about each field, see [Manifest file format (V2)](http
 }
 ```
 
+---
+
+
+<!-- ====================================================================== -->
+## Changes in manifest.json from V2 to V3
+
+| V3 | V2 |
```

Comment:
I'd prefer V2 as the first column followed by V3 because it helps to understand what changed and perform the migration instead of the current columns.